SamplePay/public/payment-manifest.json

```diff
@@ -1,8 +1,8 @@
 {
   "default_applications": [
-    "https://sample-pay-e6bb3.firebaseapp.com/manifest.json"
+    "https://sample-pay-web-app.firebaseapp.com/manifest.json"
```

Comment:
At one point I'd like to merge this into `https://bobbucks.dev/` which is a demo payment handler app I'm working on. No changes required now though.